bigtable/type.go

```diff
@@ -16,12 +16,26 @@ limitations under the License.
 
 package bigtable
 
-import btapb "cloud.google.com/go/bigtable/admin/apiv2/adminpb"
+import (
+	btapb "cloud.google.com/go/bigtable/admin/apiv2/adminpb"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
 
 // Type wraps the protobuf representation of a type. See the protobuf definition
 // for more details on types.
 type Type interface {
 	proto() *btapb.Type
+	ToJSON() (string, error)
```

Comment:
Why is this needed in this library? This seems like code that could be written where it is needed as it does not access any private data to the interface.

Reply:
It's accessing proto() that is private.
We opted to go with https://medium.com/@chaewonkong/go-and-json-a-comprehensive-guide-to-working-with-json-in-golang-143fa2dfa897#:~:text=of%20an%20integer.-,Custom%20types,-%3A instead though :)